src/Common/cmake/secure_dependencies.cmake

```diff
@@ -37,15 +37,21 @@ macro(locate_win32_spectre_static_runtime)
         find_program(_vswhere_tool
             NAMES vswhere
             PATHS "$ENV{ProgramFiles\(x86\)}/Microsoft Visual Studio/Installer")
+        message(INFO "*** _vswhere_tool: ${_vswhere_tool}")
         if (NOT ${vswhere})
             message(FATAL_ERROR "Could not locate vswhere - unable to search for installed vcruntime libraries.")
         endif()
+
         execute_process(
             COMMAND "${_vswhere_tool}" -latest -products * -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -find **/14.*.*/**/lib/spectre/x64 -sort
             OUTPUT_VARIABLE _vs_install_loc_out
             RESULT_VARIABLE _vs_where_exitcode
             OUTPUT_STRIP_TRAILING_WHITESPACE)
+        message(INFO "*** _vs_install_loc_out: ${_vs_install_loc_out}")
+
         file(TO_CMAKE_PATH "${_vs_install_loc_out}" SPECTRE_LIB_PATH_OUT)
+        message(INFO "*** SPECTRE_LIB_PATH_OUT: ${SPECTRE_LIB_PATH_OUT}")
+
```

Comment:
I assume that the changes in this file will be reverted (or commented out) before merging.

Reply:
Actually, I'm planning to keep them. They help a lot when figuring out what went wrong. I didn't have some components installed and the build was failing in a weird way - such as a function not receiving enough arguments. Turned out variables were not set earlier. By printing them out I was able to see what's wrong.